share/completions/git.fish

```diff
@@ -69,6 +73,9 @@ function __fish_git_files
     # and as a convenience "all-staged"
     # to get _all_ kinds of staged files.
 
+    # Do not continue if not inside a Git repository
+    __fish_git_is_worktree; or return
```

Comment:
Why not just 

```fish
set -l root (command git rev-parse --show-toplevel 2>/dev/null)
or return
```

?

I've never really worked with multiple work trees and such, so I'm not quite sure if `--is-inside-work-tree` is actually useful.

Reply:
Fair enough.

It also just occurred to me that `--is-inside-work-tree` will return non-zero for bare repositories, which is undesirable.